account_multicurrency_revaluation/demo/account_demo.xml

```diff
@@ -77,10 +117,18 @@
         <field name="currency_id" ref="base.GBP"/>
         <field name="default_debit_account_id" ref="demo_acc_liquidity_gbp"/>
         <field name="default_credit_account_id" ref="demo_acc_liquidity_gbp"/>
+        <field name="company_id" ref="res_company_reval" />
     </record>
     <record id="sales_journal" model="account.journal">
         <field name="name">A Sales Journal</field>
         <field name="code">SJ</field>
         <field name="type">sale</field>
+        <field name="company_id" ref="res_company_reval" />
+    </record>
+    <record id="account_payment_method_manual_in" model="account.payment.method">
+        <field name="name">Manual</field>
+        <field name="code">manual</field>
+        <field name="payment_type">inbound</field>
+        <field name="company_id" ref="res_company_reval" />
```

Comment:
remove `<field name="company_id" ref="res_company_reval" />` to make runbot happy )

Reply:
Thanks :)